SYCL/InvokeSimd/spec/simd_size/simd_size.cpp

```diff
@@ -0,0 +1,54 @@
+// TODO: enable on Windows once driver is ready
+// REQUIRES: gpu && linux
+// UNSUPPORTED: cuda || hip
+//
+// Test not intended to run on PVC
+// UNSUPPORTED: gpu-intel-pvc
+//
+// TODO: enable when Jira ticket resolved
+// XFAIL: gpu
```

Comment:
this seems messy. 2 UNSPPORTED + XFAIL which encloses them logically. Please check here and in other places that there is no unnecessary overlap in filters.

Reply:
I removed overlapping filters, but can't get rid of XFAIL till Jira issue resolved.